### PR TITLE
Insert doesn't return value

### PR DESCRIPTION
**Pull request by wild-endeavor**
# TL;DR
Insert modifies in place, doesn't return the modified value.

* Also removing the `from e` bit, it hides the stack trace which makes it super hard to investigate.

## Type
 - [x] Bug Fix
 - [ ] Feature
 - [ ] Plugin

## Are all requirements met?

 - [ ] Code completed
 - [ ] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
 _How did you fix the bug, make the feature etc. Link to any design docs etc_

## Tracking Issue
https://github.com/lyft/flyte/issues/<number>

## Follow-up issue
_NA_
OR
_https://github.com/lyft/flyte/issues/<number>_

### CODE REVIEW

**Review by kumare3 — DISMISSED** [review]

File: plugins/flytekit-greatexpectations/setup.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 microlib_name = f"flytekitplugins-{PLUGIN_NAME}"
 
-plugin_requires = ["flytekit>=0.21.0,<1.0.0", "great-expectations>=0.13.23"]
+plugin_requires = ["flytekit>=0.21.0,<1.0.0", "great-expectations==0.13.28"]
```
Comment (kumare3) [inline plugins/flytekit-greatexpectations/setup.py:7]: @samhita-alla can we actually change it atleast to be a range, with  upperbound of 0.13.28. Better would be to actually fix it for the new version, since we have not really announced anything

Comment (samhita-alla) [reply plugins/flytekit-greatexpectations/setup.py:7]: I believe https://github.com/great-expectations/great_expectations/issues/3290 is the issue.

Comment (samhita-alla) [reply plugins/flytekit-greatexpectations/setup.py:7]: We can proceed with 0.13.28 for now. I'll bump the version later.

**Review by cosmicBboy — APPROVED** [review]
